packages/core/content-releases/server/src/services/release.ts

```diff
@@ -108,6 +108,7 @@ const createReleaseService = ({ strapi }: { strapi: LoadedStrapi }) => ({
         return {
           ...release,
           action: {
+            ...actionForEntry,
             type: actionForEntry.type,
```

Comment:
If you spread all `actionForEntry` you can remove the type: actionForEntry, right?

Reply:
definitely, idk what I was thinking